src/Raven.Server/Documents/AI/AiIntegrationsController.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel.Embeddings;
+using Raven.Client.ServerWide;
+using Raven.Server.Documents.ETL.Providers.AI.Embeddings;
+using Raven.Server.Documents.ETL.Providers.AI;
+using Raven.Client.Documents.Operations.AI;
+using Raven.Client.Util;
+using Raven.Server.Documents.AI.Embeddings;
+using Sparrow.Server.Logging;
+
+#pragma warning disable SKEXP0001
+
+namespace Raven.Server.Documents.AI;
+
+public class AiIntegrationsController : IDisposable
```

Comment:
I really dislike the `Controller` suffix here, it doesn't mean what it usually does